src/modules/Search/Search.js

```diff
@@ -0,0 +1,698 @@
+import _ from 'lodash'
+import cx from 'classnames'
+import React, { PropTypes } from 'react'
+
+import {
+  AutoControlledComponent as Component,
+  customPropTypes,
+  getElementType,
+  getUnhandledProps,
+  keyboardKey,
+  makeDebugger,
+  META,
+  objectDiff,
+  useKeyOnly,
+  useValueAndKey,
+} from '../../lib'
+import { Input } from '../../elements'
+import SearchCategory from './SearchCategory'
+import SearchResult from './SearchResult'
+import SearchResults from './SearchResults'
+
+const debug = makeDebugger('search')
+
+const _meta = {
+  name: 'Search',
+  type: META.TYPES.MODULE,
+  props: {
+    size: ['mini', 'small', 'large', 'big', 'huge', 'massive'],
```

Comment:
Let's use the SUI constant here for the sizes.  It looks like all except `medium` are supported.  You can reference the Rail for example:

https://github.com/TechnologyAdvice/stardust/blob/master/src/elements/Rail/Rail.js#L41